### PR TITLE
Update project configuration to remove Jellyfish references

### PR DESCRIPTION
**Pull request by mironiasty**
Update all references in code to only use Fishjam. This include names in code, project names, build variables.
I also updated `protos`, README and few other small things

### CODE REVIEW

File: FishjamClientDemo/Release.xcconfig
Comment (karkakol) [inline FishjamClientDemo/Release.xcconfig:1]: Is this file necessary? I do not se any use of it.

File: Package.swift [hunk 1]
```diff
@@ -4,15 +4,15 @@
...
+    name: "FishjamClientSdk",
     platforms: [
         .iOS(.v13)
     ],
     products: [
         // Products define the executables and libraries a package produces, and make them visible to other packages.
         .library(
-            name: "JellyfishClientSdk",
-            targets: ["JellyfishClientSdk"])
+            name: "FishjamClientSdk",
```
Comment (karkakol) [inline Package.swift:14]: We should stick with the same name across package managers. On cocapods we have `FishjamClient` it should be named the same here.

**Review by karkakol — CHANGES_REQUESTED** [review]

File: FishjamClientDemo/Release.xcconfig
Comment (mironiasty) [reply FishjamClientDemo/Release.xcconfig:1]: It's direct usage is here: https://github.com/fishjam-dev/ios-client-sdk/pull/20/files#diff-e28b09372ab7f9778c87c348a91532b491c793208b25f9f0f7b46b7154db45ffL6
To copy as Debug configuration. I guess we can update it in future, but this would be out of scope of this PR.

File: Package.swift (hunk 1 above)
Comment (mironiasty) [reply Package.swift]: I updated it everywhere to match the naming.

**Review by karkakol — APPROVED** [review]
👍
